ci/eval/compare/default.nix

```diff
@@ -31,10 +31,10 @@ let
             changed: ["package2", "package3"],
             removed: ["package4"],
           },
-          labels: [
-            "10.rebuild-darwin: 1-10",
-            "10.rebuild-linux: 1-10"
-          ],
+          labels: {
+            "10.rebuild-darwin: 1-10": true,
+            "10.rebuild-linux: 1-10": true
+          },
```

Comment:
If I were nit-picking, I'd suggest this use the same format as the `getLabels` example; as-is this doesn't demonstrate that _all_ labels will be present as either `true` or `false` values.

Having a few `false` examples and a `# [...]` would help imply this.